spec/rails_helper.rb

```diff
@@ -28,6 +34,24 @@
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
+
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app,
+                                      js_errors: false,
+                                      phantomjs_options: ['--proxy-type=none'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.